<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyboardDialog</class>
 <widget class="QDialog" name="KeyboardDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50);</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="pushButton_Q">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Q</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_W">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>W</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_E">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>E</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_P">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>120</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>P</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_O">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>O</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_I">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>120</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>I</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_U">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>U</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Y">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_T">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>T</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_R">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>R</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_D">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>D</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_L">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>190</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>L</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_J">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>J</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_G">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>G</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_S">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>S</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_F">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>F</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_A">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_H">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>H</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_K">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>K</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_N">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>N</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_M">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>260</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>M</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_C">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_X">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_B">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_V">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>V</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Z">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Space">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>490</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ChangeKeyboard">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>123</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Enter">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>70</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(50, 200, 50, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputField">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225, 225, 225);</string>
   </property>
   <property name="text">
    <string>Input</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>481</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(199, 199, 199);</string>
   </property>
   <property name="text">
    <string>Enter beer name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Shift">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>80</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Shift</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Delete">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>270</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(200, 50, 50, 100);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
